modules/auxiliary/scanner/mysql/mysql_login.rb

```diff
@@ -5,13 +5,14 @@
 
 require 'metasploit/framework/credential_collection'
 require 'metasploit/framework/login_scanner/mysql'
+require 'rex/post/mysql'
```

Comment:
is this needed?

Reply:
No seems it wasn't. I think I had added this when trying to get things working and forgot to remove it. Good catch 👍

Reply:
Looks like this is back again? Is the latest code pushed up here @cgranleese-r7 ?

Reply:
The changes hadn't been pushed up just yet. Sorry about the confusion.